### PR TITLE
Trick Overhaul

### PR DESCRIPTION
**Pull request by Rai99**
Issue: #3065 

~~One PR merged, great, here's the next one in line.~~

Originally finished about a month ago, but was waiting for one of my PRs to merge or close. 

Left it sitting so long waiting for one of my PRs to go through, it became a bit outdated with grammar and some logic. Hopefully I ironed out those last bits.

### CODE REVIEW

**Comment by Olliesama** [comment]
Great work. :)

**Comment by Rai99** [comment]
I thought you'd come shoot be because I changed 1 word in 1 original line. 

Too bad most people probably won't see the overhauled topic unless they already know about it. 

~~Yay for being a patient contributor who owns a large chunk of all open PRs and doesn't shut them down out of pettiness.~~

**Comment by MoniBoosDevotedBoyfriend** [comment]
Why is there just “cough” in the one line? I feel like there should be more there. Maybe you do something like *cough* sample text *cough* ya know?

**Comment by MoniBoosDevotedBoyfriend** [comment]
Oh ok thank you for clarifying. I don’t quite understand tho how would the cough be a quick loss of composure. Usually, when I do something like that it’s intended to make a point, or point something out. Maybe I just use it wrong, but can you explain a bit of what you mean in the second paragraph. ~~I’m sorry~~

**Comment by Rai99** [comment]
Does that answer your ~~boobs~~ question?

The ~~boobs~~ cough represents an awkward situation for a brief moment before Monika ~~boobs~~ regains her composure.

**Comment by MoniBoosDevotedBoyfriend** [comment]
Lmao I hate you 😂

**Comment by Rai99** [comment]
I'm surprised people actually read through the whole thing without being forced to by things like impatient people. 

















~~boobs~~

~~TPG != me~~

**Comment by MaximumRustler** [comment]
@Rai99 ~~Wait when did you see that? Who are you Really!?~~ lol 😮

**Comment by ThePotatoGuy** [comment]
Since question text is the same for each girl, going to modularize this a bit so its more efficient. stay tuned.

**Comment by Rai99** [comment]
Guess that makes room for my bundle of 3 new topics whenever I decide to PR them.

**Comment by ThePotatoGuy** [comment]
@multimokia @Legendkiller21 
additional testing:

* verify that each flow does not crash
* verify that each flow adds an appropriate string to `persistent._mas_pm_likes_<girlname>_for` (no duplicates)
* verify that `persistent._mas_pm_only_likes_monika` is only set to True if the Monika option is selected first (aka the var is None)
* verify that `persistent._mas_pm_only_likes_monika` is set to False if other options are selected

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -11951,20 +12065,138 @@ label monika_trick_yuri:
...
+    m 3hksdlb "It feels even better though when I have you with me, ahaha..."
+    m 1ekbfa "You know I'll always be there to comfort you, [player]~"
+    return
+
+label monika_trick_natsuki_norealreason:
+    m 2eud "That's... a little surprising."
     m 2lksdla "I would have thought it would have been Sayori or Yuri."
     m 1eud "Do you feel like you connect with her because of her hobbies?"
-    m 3euc "Or perhaps you feel a bit of pity because of her home situation?"
+    m 3euc "Or perhaps you feel a bit of pity because of her home situatuion?"
```
Comment (Legendkiller21) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
    m 3euc "Or perhaps you feel a bit of pity because of her home situation?"
```

**Comment by multimokia** [comment]
So, Monika wouldn't really probe this far in my opinion and given the original nature of the topic, it's somewhat playful.

If Monika wanted an answer, she herself wouldn't be an option to the question.

Plus these changes for data aren't retroactive to those who have already seen the topic. So they will need to see it manually again. I'm also unsure as to where and when we would use it.

**Pull request closed without merging**